src/zkevm_specs/evm/table.py

```diff
@@ -38,6 +40,62 @@ class FixedTableTag(IntEnum):
     StackOverflow = auto()  # opcode, stack_pointer, 0
     StackUnderflow = auto()  # opcode, stack_pointer, 0
 
+    def table_assignments(self) -> Sequence[Array4]:
+        if self == FixedTableTag.Range16:
+            return [(self, i, 0, 0) for i in range(16)]
+        elif self == FixedTableTag.Range32:
+            return [(self, i, 0, 0) for i in range(32)]
+        elif self == FixedTableTag.Range64:
+            return [(self, i, 0, 0) for i in range(64)]
+        elif self == FixedTableTag.Range256:
+            return [(self, i, 0, 0) for i in range(256)]
+        elif self == FixedTableTag.Range512:
+            return [(self, i, 0, 0) for i in range(512)]
+        elif self == FixedTableTag.Range1024:
+            return [(self, i, 0, 0) for i in range(1024)]
+        elif self == FixedTableTag.SignByte:
+            return [(self, i, (i & 1) * 0xFF, 0) for i in range(256)]
```

Comment:
Shouldn't this be like this, in order to check the most significant bit (which tells the signed-ness in two's complement)?:
```
return [(self, i, ((i >> 7) & 1) * 0xFF, 0) for i in range(256)]
```

Reply:
Ah you are right. Thanks for catching this, it's fixed in [`13d7187`](https://github.com/appliedzkp/zkevm-specs/pull/88/commits/13d7187364cf8ba66bfbea3f405782af8d49345e)